src/foam/nanos/boot/Boot.java

```diff
@@ -156,7 +156,7 @@ public Object create(X x) {
     ((ProxyDAO) root_.get("nSpecDAO")).setDelegate(
       new foam.nanos.auth.AuthorizationDAO.Builder(getX())
         .setDelegate(serviceDAO_)
-        .setAuthorizer(new foam.nanos.auth.GlobalReadAuthorizer("service"))
+        .setAuthorizer(new foam.nanos.auth.AuthorizableAuthorizer("service"))
```

Comment:
Couldn't we just use a StandardAuthorizer, and then avoid making the NSpec Authorizable? Is there any custom logic that is required?

Reply:
the read uses custom logic which checks authenticate and a different permission from the standardauthorizer but the other methods do not need custom logic. created a pr to address https://github.com/kgrgreer/foam3/pull/1598